### ISSUE TITLE
Re-consider font choice

### ISSUE THREAD

**Issue by marcoow** [mainmatter/mainmatter.com#559]
We are currently using the Nunito Sans font which I picked more or less randomly at some point. As that is obviously not a decision that is backed by any expertise with the topic of fonts at all, we should maybe reconsider (which could of course also result in sticking with the current choice).

I think requirements for a font are (please extend or correct this list):

- must be readable on all kinds of devices, ideally printed as well so we could use that one font everywhere
- must be available for use on all devices that we want to support (not sure that's not the case for all fonts anyway?)
- must either be free or reasonably priced
- must fit in with our general branding (which I think should mainly stand for simplicity, modern-ness)

### TODO

- [ ] decide on a font
- [ ] if it's not Nunito, switch

**Comment by pichfl** [comment]
The current design and it's typographic rhythm is based on Nunito. 

Picking a different font might as well be a task for reiterating the basis of the current design. Speaking as a designer, I'm always in favor of this, but speaking as someone who just barely finished the current iteration of the website, I'm not sure we should do this right now.

Nunito Sans is a full fledged font family and was extended to work in all situations, so I don't have any objections when it comes to readability. Nunito Sans feels feels like it drew a lot of inspiration from Proxima Nova, which has dominated parts of the design scene in recent years. Yes, modern and yes, maybe part of a fad, but still elegant. 

TLDR; There is no reason to question your choices unless you're not happy with how everything looks right now.

**Comment by marcoow** [comment]
ok, perfect 👍
